.appveyor.yml

```diff
@@ -52,19 +50,13 @@ install:
   - conda config --prepend channels conda-forge
 
   # For building, use a new environment
-  - conda create -q -n test-environment python=%PYTHON_VERSION% tk "pip<22.0"
-  - activate test-environment
+  - echo ^ ^ - python=%PYTHON_VERSION% >> environment.yml
+  - conda env create -f environment.yml
+  - activate mpl-dev
```

Comment:
Is it not `conda activate mpl-dev` now?

Reply:
I think either works perfectly fine, I just changed the name not the command, because it was still a conda environment that was created before.

Reply:
Maybe it's just the `source activate` version that is deprecated, but if it works, I guess we should migrate to the new standard one.

Reply:
actually, `conda activate` (and the suggestion from the error message of `CALL conda.bat activate`) do not work